packages/react/README.md

```diff
@@ -81,3 +81,32 @@ const RoundButton = styled(Button)({
   borderRadius: borderRadius('full'),
 });
 ```
+
+## Use for prototyping
+
+If you need to prototype with Castor, the quickest way is to use the generated single file global `Castor` exposed as UMD module:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@onfido/castor-react/prototype/react.min.js"></script>
```

Comment:
How do you feel about just naming the file `prototype.js` and putting it in the root of each package?

`@onfido/castor-react/prototype.min.js`

That way the import doesn't include the library name twice, `react` in this case.